pegasus/sites.v3/code.org/public/css/homepage.css

```diff
@@ -354,13 +362,15 @@ body.modal-open .supreme-container{
   height: 40px;
   padding: 0 30px;
   margin: 10px 5px 0 0;
+  background-color: var(--orange);
+  border-color: var(--orange);
```

Comment:
Keeping this button orange for now since it's used on the default homepage hero banner.